specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json

```diff
@@ -0,0 +1,2608 @@
+            "RSA_OAEP",
+            "PlainText"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SupportedAlgorithm",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "DataManager": {
+      "description": "The DataManager resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "etag": {
+          "description": "Etag of the Resource.",
+          "type": "string"
+        }
+      }
+    },
+    "DataManagerList": {
+      "description": "DataManager resources Collection.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of data manager resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataManager"
+          }
+        },
+        "nextLink": {
+          "description": "Link for the next set of data stores.",
+          "type": "string"
+        }
+      }
+    },
+    "DataService": {
+      "description": "Data Service.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DmsBaseObject"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/DataServiceProperties",
+          "description": "DataService properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DataServiceList": {
+      "description": "Data Service Collection.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of data services.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataService"
+          }
+        },
+        "nextLink": {
+          "description": "Link for the next set of data services.",
+          "type": "string"
+        }
+      }
+    },
+    "DataServiceProperties": {
+      "description": "Data Service properties.",
+      "required": [
+        "state"
+      ],
+      "type": "object",
+      "properties": {
+        "state": {
+          "description": "State of the data service.",
+          "enum": [
+            "Disabled",
+            "Enabled",
+            "Supported"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "State",
+            "modelAsString": false
+          }
+        },
+        "supportedDataSinkTypes": {
+          "description": "Supported data store types which can be used as a sink.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "supportedDataSourceTypes": {
+          "description": "Supported data store types which can be used as a source.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DataStore": {
```

Comment:
The 200 response model for an ARM PUT operation must have x-ms-azure-resource extension set to true in its hierarchy

Reply:
Waiting for a suppression on this issue.

Reply:
Suppression added